parlai/agents/seq2seq/modules.py

```diff
@@ -692,7 +692,7 @@ def forward(self, xes, hidden, attn_params):
             # calculate activation scores, apply mask if needed
             if attn_mask is not None:
                 # remove activation from NULL symbols
-                attn_w_premask.masked_fill_((~attn_mask), -NEAR_INF)
+                attn_w_premask.masked_fill_((~attn_mask), neginf(enc_out.dtype))
```

Comment:
nit; would it be slightly safer to use `neginf(attn_w_premask.dtype)`? Or is it necessarily the case that we want `enc_out` and `attn_w_premask` to have the same dtype?

Reply:
Yeah you're right.